docs/source/pages/howToWorkWithPlatform/mainApplication/templateList/templateManagement/documentManagementOnTemplate/documentManagement.rst

```diff
@@ -2,13 +2,13 @@
 Document management
 ===================
 
-This section describes all functionality of document editor. Document editor page contain the following data:
+This section describes all functionality of document editor. Please note that amount of documents per envelope is limited to 50 by default system configuration. Document editor page contain the following data:
```

Comment:
We should clarify that limit of 50 documents applies only for External documents